src/main/java/Duke.java

```diff
@@ -1,10 +1,159 @@
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    public static String h_rule = "────────────────────────────────────────────────────────────────────";
```

Comment:
Perhaps h_rule can be renamed as HORIZONTAL_LINE_RULE. Perhaps it can be made a constant and hence named in all caps.